commons-api/src/main/java/org/exoplatform/commons/api/notification/model/MessageInfo.java

```diff
@@ -170,6 +193,7 @@ public Message makeEmailNotification() {
     message.setTo(to);
     message.setSubject(subject);
     message.setBody(body + ((footer != null && footer.length() > 0) ? footer : ""));
+    this.attachments.stream().forEach(message::addAttachment);
```

Comment:
Hi @rdenarie, i'm getting a npe in this point, see: 
![image](https://user-images.githubusercontent.com/103621472/164306735-11a2e2f6-616d-448c-8a92-e65bc7564095.png)
Maybe we can do a null check before the stream right?

Reply:
Hello.
Tahnks for your contribution.
Of course a null test is needed here.
Are you available to make a PR ?